web/styles/modal.css

```diff
@@ -367,8 +367,12 @@
     }
 }
 
-.modal_select {
+#user-role-select,
+#bot-role-select {
```

Comment:
I don't understand how a height of `30px` is going to adjust with info density at all?

Reply:
It's been a while, but I think the idea was that if `modal_select` doesn't have a fixed height then it can get it from the text size, but the user role and bot role select still needed the 30px height for some reason.